added: Output variables section for upload to AWS s3 step (#1466)
OctopusDeploy/docs: docs/deployments/aws/s3/index.md

This behavior will be configurable in a future release to provide additional flexibility.
:::

### Output variables

> Once the file(s) is uploaded, Octopus will create several variables storing the properties of the uploaded files. These values can be used in subsequent steps to access the uploaded files.

Reviewer: ```suggestion
Octopus will create several output variables storing the properties of any uploaded files. The values can then be used in subsequent steps to access the uploaded files.
```
